assets/sass/lbd/_pages.scss

```diff
@@ -127,6 +126,10 @@
     .btn-wd{
       min-width: 180px;
     }
+
+    .card-header {
```

Comment:
Will this affect all cards in the application?

Reply:
Yes, but it's a small change and I didn't see any impact on the already implemented cards.
The thing is that with dark backgrounds the corners didn't look well, so I adjusted them like this.